crates/agentgateway/src/proxy/httpproxy.rs

```diff
@@ -484,10 +484,6 @@ impl HTTPProxy {
 			log.outgoing_span = Some(ns);
```

Comment:
It looks like we're setting `log.outgoing_span` here before we fetch frontend policies and override the `log.tracer`. Do we want this span to be created with the static trace, or do we want to move the dynamic tracer definition earlier and use that? If we use the static tracer here, the user-configured exporter/attrs/sampler won’t be used for this request’s span.